plutus-core/plutus-core/src/PlutusCore/Builtin/KnownType.hs

```diff
@@ -352,6 +353,8 @@ instance
         , uni ~ UniOf val
         ) => ReadKnownIn uni val (EvaluationResult a) where
     readKnown _ = throwing _UnliftingError "Panic: 'TypeError' was bypassed"
+    -- Just for 'readKnown' not to appear in the generated Core.
```

Comment:
What problem does it cause if `readKnown` appears in Core?

Reply:
Added this:

```
Neither 'readKnown' nor 'makeKnown' should appear in the generated Core for builtins. In most cases
they would slow builtins down, but even if a 'readKnown' only throws an error, it still makes sense
to keep it out of Core just not to trigger an investigation on whether it's fine that a call to
'readKnown' is not inlined.
```

to `Note [Performance of ReadKnownIn and MakeKnownIn instances]`.